plugins/by-name/chadtree/default.nix

```diff
@@ -502,11 +498,21 @@ in
       };
     in
     mkIf cfg.enable {
-      extraPlugins =
-        [ cfg.package ]
-        ++ (optional (
-          cfg.iconsPackage != null && (cfg.theme == null || cfg.theme.iconGlyphSet == "devicons")
-        ) cfg.iconsPackage);
+      # TODO: added 2024-09-20 remove after 24.11
+      plugins.web-devicons.enable = mkOverride 1490 true;
+      warnings =
+        optional
+          (
+            (cfg.theme == null || cfg.theme.iconGlyphSet == "devicons")
```

Comment:
If this is true when `cfg.theme == null`, perhaps it should also be true when `cfg.theme.iconGlyphSet == null`?

Reply:
E.g. `elem cfg.theme.iconGlyphSet or null [ null "devisons" ]`

Reply:
I was just wanting to show the warning for the condition that used to auto install the package.